src/coverage.ts

```diff
@@ -77,14 +138,27 @@ let coverageStatusBarObject: vscode.StatusBarItem;
 function resetGlobalDecorations() {
   uncoveredDecorations = [];
   coveredDecorations = [];
+  partiallyCoveredDecorations = [];
```

Comment:
try packing decoration types and decorations into 2 lists, then iterating and reseting each element of a list with a loop

Reply:
Refactored in ca80e6c05de81bc3fda70e854b82c0cee7479e9a.